src/System.Reflection.TypeExtensions/tests/MemberInfo/MemberInfoMemberType.cs

```diff
@@ -1,214 +0,0 @@
-    // the type of the member  method, constructor, event, and so on. 
-    public class ReflectionMemberInfoMemberType
-    {
-        // PosTest1: get accessor of public static property
-        [Fact]
-        public void PosTest1()
-        {
-            bool expectedValue = true;
-            bool actualValue = false;
-            MethodInfo methodInfo;
-            MemberInfo memberInfo;
-            methodInfo = typeof(TestClass1).GetProperty("InstanceCount").GetGetMethod();
-            memberInfo = methodInfo as MemberInfo;
-            actualValue = memberInfo.Name.Equals("get_InstanceCount");
-            Assert.Equal(expectedValue, actualValue);
-        }
-
-        // PosTest2: set accessor of public instance property
-        [Fact]
-        public void PosTest2()
-        {
-            bool expectedValue = true;
-            bool actualValue = false;
-            MethodInfo methodInfo;
-            MemberInfo memberInfo; methodInfo = typeof(TestClass1).GetProperty("Data1").GetSetMethod();
-            memberInfo = methodInfo as MemberInfo;
-            actualValue = memberInfo.Name.Equals("set_Data1");
-            Assert.Equal(expectedValue, actualValue);
-        }
-
-        // PosTest3: public static property 
-        [Fact]
-        public void PosTest3()
-        {
-            bool expectedValue = true;
-            bool actualValue = false;
-            PropertyInfo propertyInfo;
-            MemberInfo memberInfo;
-            propertyInfo = typeof(TestClass1).GetProperty("InstanceCount");
-            memberInfo = propertyInfo as MemberInfo;
-            actualValue = memberInfo.Name.Equals("InstanceCount");
-            Assert.Equal(expectedValue, actualValue);
-        }
-
-        // PosTest4: public instance property
-        [Fact]
-        public void PosTest4()
-        {
-            bool expectedValue = true;
-            bool actualValue = false;
-            PropertyInfo propertyInfo;
-            MemberInfo memberInfo;
-            propertyInfo = typeof(TestClass1).GetProperty("Data1");
-            memberInfo = propertyInfo as MemberInfo;
-            actualValue = memberInfo.Name.Equals("Data1");
-            Assert.Equal(expectedValue, actualValue);
-        }
-
-
-        // PosTest5: public constructor 
-        [Fact]
-        public void PosTest5()
-        {
-            bool expectedValue = true;
-            bool actualValue = false;
-            ConstructorInfo constructorInfo;
-            MemberInfo memberInfo;
-            Type[] parameterTypes = { typeof(int) };
-            constructorInfo = typeof(TestClass1).GetConstructor(parameterTypes);
-            memberInfo = constructorInfo as MemberInfo;
-            actualValue = memberInfo.Name.Equals(".ctor"); ;
-            Assert.Equal(expectedValue, actualValue);
-        }
-
-        // PosTest6: private instance field
-        [Fact]
-        public void PosTest6()
```

Comment:
Sorry, should have mentioned.
They exist [here](https://github.com/dotnet/corefx/blob/master/src/System.Reflection.TypeExtensions/tests/FieldInfoTests.cs#L15) and [here](https://github.com/dotnet/corefx/blob/master/src/System.Reflection.TypeExtensions/tests/FieldInfoTests.cs#L20)